main.go

```diff
@@ -119,11 +120,11 @@ func execute(dbprovider string, excludeNamespaces, includeNamespaces []string, r
 				if excluded(db, excludeNamespaces, includeNamespaces) {
 					return
 				}
-				client := client.CrdClient(crdcs, scheme, db.Namespace) // add the database namespace to the client
-				err = handleCreateDatabase(db, client, dbprovider, repository)
+				_client := client.CrdClient(crdcs, scheme, db.Namespace) // add the database namespace to the client
```

Comment:
same here with the _

Reply:
@sorenmat 
client variable collides with imported package `"github.com/sorenmat/k8s-rds/client"`,